### PR TITLE
[nextjs-react] Update to use @ syntax for remotes

### PR DESCRIPTION
**Pull request by JohnDaly**
## Changes
- Updating `nextjs-react` example to use the @ syntax for remotes
- Updating `nextjs-react` example to use `yarn` instead of `npm`. 
  - There was mixed usage of both in this example. I opted for `yarn` because it is used in more examples throughout the monorepo

## Other Fixes
- Adding `.npmrc` file to set privjs as registry for `@module-federation` packages
- Updating ESLint config in `nextjs-react/host-app` to fix linter warnings about `next/babel`
- Updating `webpack-cli` to fix `TypeError: cli.isMultipleCompiler is not a function` error
- Updating `webpack` and `webpack-dev-server` to latest versions

### CODE REVIEW

File: nextjs-react/remote-app/package.json [hunk 1]
```diff
@@ -8,9 +8,9 @@
     "babel-loader": "8.2.5",
     "html-webpack-plugin": "5.5.0",
     "serve": "13.0.2",
-    "webpack": "5.72.1",
-    "webpack-cli": "4.9.2",
-    "webpack-dev-server": "4.8.1"
+    "webpack": "5.73.0",
+    "webpack-cli": "4.10.0",
```
Comment (JohnDaly) [inline nextjs-react/remote-app/package.json:12]: Fixes:

```
[0] [webpack-cli] TypeError: cli.isMultipleCompiler is not a function
[0]     at Command.<anonymous> (/module-federation-examples/nextjs-react/remote-app/node_modules/@webpack-cli/serve/lib/index.js:146:35)
[0]     at async Promise.all (index 1)
[0]     at async Command.<anonymous> (/module-federation-examples/nextjs-react/remote-app/node_modules/webpack-cli/lib/webpack-cli.js:1674:7)
```

File: nextjs-react/host-app/.eslintrc.json [hunk 2]
```diff
@@ -1,3 +1,3 @@
 {
-  "extends": "next/core-web-vitals"
+  "extends": ["next/babel", "next/core-web-vitals"]
```
Comment (JohnDaly) [inline nextjs-react/host-app/.eslintrc.json:2]: Fixes the `Parsing error: Cannot find module 'next/babel'` error from ESLint

**Review by ScriptedAlchemy — CHANGES_REQUESTED** [review]
Please check the v4 release for nextjs-mf 

The api has changed slightly. No need to double configure MFP on next apps. All is done with the sidecar. Ideally I'd use the new major release if you're moving to the @ syntax

**Comment by JohnDaly** [comment]
> Please check the v4 release for nextjs-mf
> 
> The api has changed slightly. No need to double configure MFP on next apps. All is done with the sidecar. Ideally I'd use the new major release if you're moving to the @ syntax

I've updated to the v4 release and updated the Next.js host app in the example to use `withFederatedSidecar` in https://github.com/module-federation/module-federation-examples/pull/2067/commits/ec84bdf38e1743a11fe20db03d0055ad29ee078d

**Comment by ScriptedAlchemy** [comment]
Perfect. Will review tomorrow morning

**Comment by JohnDaly** [comment]
@ScriptedAlchemy do the changes look good to you?

**Pull request merged**